### ISSUE TITLE
Finetuning on specific datasets

### ISSUE THREAD

**Issue by HashmatShadab** [NVlabs/EAGLE#10]
Is there an option in the codebase to do the finetuning on only selected datasets mentioned in the readme?

**Comment by flyinglynx** [comment]
Yes, you can convert your dataset into LLaVA's format and update the data path in the script accordingly.

Essentially, you'll need to transform your annotations into a list of conversation data. For more details, please refer to the example annotation JSON file. If you run into any issues, feel free to reach out to us.

If you dataset is small, please consider using efficient tuning techniques like LoRA.

**Comment by HashmatShadab** [comment]
Thank you for explaining. I was specifically talking about using specific datasets for finetuning that are mentioned in the readme. So for that i can just update the json file
